package.json

```
@@ -10,15 +10,19 @@
     "build": "babel src -s -D -d dist",
     "start": "pm2-dev --interpreter babel-node src",
     "start:dev": "npm run migrate && npm start",
-    "serve": "pm2 start dist"
+    "serve": "pm2 start dist",
+    "seed": "knex seed:run"
```

```suggestion
    "seed": "node -r dotenv/config ./node_modules/knex/bin/cli.js seed:run"
```
This will load .env values before knex seed